developers/weaviate/manage-data/index.md

```diff
@@ -19,7 +19,6 @@ The guides cover these topics:
 - [Cross-references](./cross-references.mdx): Manage references between collections.
 - [Multi-tenancy operations](./multi-tenancy.md): Operational differences when multi-tenancy is enabled.
 
-import DocsMoreResources from '/_includes/more-resources-docs.md';
+import DocsMoreResources from '/\_includes/more-resources-docs.md';
```

Comment:
Could you please revert this back?

Reply:
I didn't notice when the VS code formatter added that. Thanks for letting me know.